{
    "bug_type": "NULL_DEREFERENCE",
    "qualifier": "pointer `null` is dereferenced by call to `Context(...)` at line 38, column 13.",
    "severity": "ERROR",
    "line": 38,
    "column": 13,
    "procedure": "Void ContextUTTests.TestCqlFromContext()",
    "procedure_start_line": 37,
    "file": "src/Cassandra.IntegrationTests/Linq/ContextUTTests.cs",
    "bug_trace": [
        {
            "level": 0,
            "filename": "src/Cassandra.IntegrationTests/Linq/ContextUTTests.cs",
            "line_number": 37,
            "column_number": 9,
            "description": "start of procedure TestCqlFromContext()"
        },
        {
            "level": 0,
            "filename": "src/Cassandra.IntegrationTests/Linq/ContextUTTests.cs",
            "line_number": 38,
            "column_number": 13,
            "description": ""
        }
    ],
    "key": "ContextUTTests.cs|TestCqlFromContext|NULL_DEREFERENCE",
    "node_key": "af4afc3f16bbd4d47126044f9dac65e9",
    "hash": "8798b32a25ecbeb528025297992e2b21",
    "bug_type_hum": "Null Dereference"
}